.github/workflows/network-api.yml

```
@@ -0,0 +1,23 @@
+
+  deploy:
+    name: ????
+    needs: test
+    uses: ./.github/workflows/deploy.yml
+    secrets:
+      HOST: ${{ secrets.NETWORK_API_ADDRESS }}
+      USERNAME: ${{ secrets.NETWORK_API_USERNAME }}
+      PASSWORD: ${{ secrets.NETWORK_API_PASSWORD }}
+      SSH_PORT: ${{ secrets.NETWORK_API_SSH_PORT }}
+    with:
+      environment: network-api
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)